index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio Website</title>
+</head>
+<body>
+    <header>
+        <div class="logo">
+          <a>My Logo</a>
+        </div>
+        <nav class="nav-bar">
+            <div class="ham-menu">
+            <img src="image/Vector.png" alt="Nav Bar" class="navImg1">
+            <img src="image/Vector.png" alt="Nav Bar" class="navImg2">
+            <img src="image/Vector.png" alt="Nav Bar" class="navImg3">
+        </div>
+        </nav>
+    </header>
+    <main>
+       <section>
+           <div class="Backgroud">
+  
+    
+           </div>
```

Comment:
- [ ] Instead of using a `div` tag inside of the parent container to represent your background image, why not use the parent container and apply a background image to it. Kindly implement this.